Bugzilla/Bug.pm

```diff
@@ -302,6 +302,73 @@ with 'Bugzilla::Elastic::Role::Object';
 
 sub ES_TYPE {'bug'}
 
+sub ES_INDEX { Bugzilla->params->{elasticsearch_index} }
+
+sub ES_SETTINGS {
```

Comment:
Pretty much copied over. looking at this, I think I should remove two of the analyzers that are only used in the users index... but that opens the can of worms about when the index settings don't match the code -- which should be addressed in a different bug.